tests/flows/validator-manager/erc20_delegation.go

```diff
@@ -46,17 +45,17 @@ func ERC20Delegation(network interfaces.LocalNetwork) {
 		pChainInfo,
 	)
 
-	_ = utils.InitializeERC20TokenValidatorSet(
-		ctx,
-		fundedKey,
-		subnetAInfo,
-		pChainInfo,
-		stakingManager,
-		stakingManagerAddress,
-		network,
-		signatureAggregator,
-		utils.DefaultMinStakeAmount*10,
-	)
+	// _ = utils.InitializeERC20TokenValidatorSet(
```

Comment:
Why is this getting commented out?

Reply:
This is one of a series of PRs (https://github.com/ava-labs/teleporter/pull/585 and https://github.com/ava-labs/teleporter/pull/592) that touch individual parts of the contracts and testing components. Rather than getting each to pass CI individually, I elected to stub/comment out incompatible code. The final PR in the series has everything intact. The focus for this PR is really only `erc20_staking.go`